README.md

```diff
@@ -77,3 +78,65 @@ In order to expose those ports outside the local environment, you can edit lines
       - "${SCNODE_WS_SERVER_PORT}:${SCNODE_WS_SERVER_PORT}"
       - "${SCNODE_REST_PORT}:${SCNODE_REST_PORT}"
    ```
+
+## Running a Forger Node 
+Should you want to run a forger node, please select "forger" when you run the init.sh script for the first time. Keep in mind it will take time for the Mainchain node and the sidechain node to fully sync the whole chains.
+
+Step by step guide : 
+
+1 - On init.sh script select "forger" when prompted. 
+2 - Once the Evm node and the Mainchain node are up&running, run the generate_forger_keys script 
```

Comment:
Shall we mention that this step requires to nodes to be fully synced as well? Perhaps something like this

> Once EVM and Mainchain nodes are up and fully synced with the blockchain, run the 'generate_forger_keys.sh' script.

Reply:
Actually you don't need the nodes to be fully synced in order to generate those keys ; you will need the nodes to be fully synced in order to forge a block. I'll add something about his.